public/locales/en/common.json

```diff
@@ -10,6 +10,7 @@
   "contactUsHeading": "Contact Us",
   "contactUsContent": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus consectetur urna nec tempor tristique.",
   "featuredPrototypesHeading": "Featured prototypes",
+  "dateModified": "Date Modified:",
```

Comment:
We prefer [sentence case (vs title case)](https://titlecaseconverter.com/blog/sentence-case-vs-title-case/), so "Date modified" not "Date Modified"

Reply:
@pcraig3 should I change the other values like "Contact Us" to "Contact us" while I'm at it?

Reply:
Yes please 👍

Reply:
Values changed